.bazelrc

```diff
@@ -1,7 +1,6 @@
 test --announce_rc
 test --test_output=errors
 test --test_verbose_timeout_warnings
-test --spawn_strategy=linux-sandbox
```

Comment:
Iirc I added this to get an explicit build error rather than the worker falling back to a less hermetic execution: https://bazel.build/docs/sandboxing

I agree `sandboxed` makes most sense, let's go with that.